source/FreeRTOS_DNS.c

```diff
@@ -1082,9 +1082,7 @@
         /* MISRA Ref 11.3.1 [Misaligned access] */
         /* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#rule-113 */
         /* coverity[misra_c_2012_rule_11_3_violation] */
-        const DNSMessage_t * pxDNSMessageHeader =
-            ( ( const DNSMessage_t * )
-              pxReceiveBuffer->pucPayloadBuffer );
+        const DNSMessage_t * pxDNSMessageHeader = ( ( const DNSMessage_t * ) pxReceiveBuffer->pucPayloadBuffer );
```

Comment:
Can the extra brackets be removed?

~~~diff
-const DNSMessage_t * pxDNSMessageHeader = ( ( const DNSMessage_t * ) pxReceiveBuffer->pucPayloadBuffer );
+const DNSMessage_t * pxDNSMessageHeader = ( const DNSMessage_t * ) pxReceiveBuffer->pucPayloadBuffer;
~~~